### PR TITLE
#8 & #10

### PR DESCRIPTION
**Pull request by hakancelik96**

### CODE REVIEW

File: tests/samples/configs/setup.cfg [hunk 1]
```diff
@@ -18,6 +18,7 @@ max-line-length = 119
 statistics = True
 
 [unimport]
-folders = .*(migrations)
-files = .*(__init__.py)
-        .*(settings.py)
+ignore = 
+        ./[0-9].*
+        tests
+
```
Comment (isidentical) [inline tests/samples/configs/setup.cfg]: nit: EOF fix

File: unimport/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+__description__ = "Detect or remove unused Python imports."
+__version__ = "0.1.0"
+__author__ = (
```
Comment (isidentical) [inline unimport/__init__.py]: this should be a string not a tuple, "John Doe, Hakan Celik" ...

Comment (hakancelik96) [reply unimport/__init__.py]: I write like this because of I saw this style at io.

https://github.com/python/cpython/blob/master/Lib/io.py#L37

Comment (isidentical) [reply unimport/__init__.py]: kral o tuple degil implicit string concat
```
    "Hakan Çelik <hakancelik96@outlook.com>, ",
    "Batuhan Taşkaya <>",
    "Gökmen Görgen <>",
```
yerine
```
    "Hakan Çelik <hakancelik96@outlook.com>, "
    "Batuhan Taşkaya <>, "
    "Gökmen Görgen <>"
```
yapmalisin (disarda virgul olunca tuple oluyor)

**Pull request merged**